packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1165,6 +1174,14 @@ class SemanticsProperties extends DiagnosticableTree {
   /// [maxValueLength] is set.
   final int? currentValueLength;
 
+  /// Provides an identifier for UI testing. This value is not exposed to the
+  /// user.
+  ///
+  /// On Android, this will become the node's `resource-id`.
```

Comment:
There is no resource id in AccessibilityNodeInfo

Reply:
`resource-id` is in the native accessibility hierarchy in XML that can be obtained with `adb shell uiautomator dump`. I'll also mention the relevant property in `AccessibilityNodeInfo` (`viewIdResourceName`).

Reply:
[I'm assuming this is resolved](https://github.com/flutter/flutter/pull/138331#pullrequestreview-1737575589). Feel free to un-resolve in case of more questions.